pyanaconda/modules/storage/devicetree/viewer.py

```diff
@@ -117,7 +117,7 @@ def _set_device_data(self, device, data):
         data.name = device.name
         data.path = device.path
         data.size = device.size.get_bytes()
-        data.parents = [d.name for d in device.parents]
+        data.parents = [getattr(d, "id_path", d.name) for d in device.parents]
```

Comment:
We use the device name as a unique identifier that allows to traverse the device tree. Providing something different than the id could have dangerous side effects. If we need this just to display different info about the parents, we could collect the parent device data in the UI and modify the UI part.

Reply:
Got it.
I removed the dangerous change (and thus the actual UI change) with 20bbea9 [temporary squash commit so you can easily see the differential change; I can squash things before a merge].
Unfortunately, I don't know nor understand the UI mechanics, so I am not able to modify the UI display to show path-id (if available) instead of kernel device names for the multipath members.
But at least, zfcp now also provides the necessary base data with the path-id attribute as a pre-req for a potential and separate UI change.

Reply:
I think this is what ends up in the UI column "Paths":
https://github.com/rhinstaller/anaconda/blob/ae5038190264f6bec2b95a8c4f862cf2e9edbc48/pyanaconda/ui/gui/spokes/advanced_storage.py#L85

Reply:
Yeah, I think the team can handle this change in a separate pull request, so I wouldn't worry about this part.